src/openzaak/js/components/admin/applicatie-autorisaties/extra-attributes.js

```diff
@@ -114,13 +133,13 @@ const ExternalTypes = (props) => {
 };
 
 
-const TypeOptions = (props) => {
+const CatalogusTypeOptions = (props) => {
     const { typeOptionsField, selectedValues, externalValues, onChange, errorsExternal } = props;
     const catalogi = useContext(CatalogiContext);
     return (
         <Fragment>
             { catalogi.map(catalogus => (
-                <CatalogusOptions
+                <TypeOptions
```

Comment:
sounds like a good idea :+1:

Reply:
I've checked if I could manage to do with with react-select, but it's taking me too much time. I think it might be better to tackle this in a separate issue instead of doing it here

Reply:
Sure, but please create an issue for it